tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -0,0 +1,1321 @@
+            duthost.shell(proxy_arp_del_cmd.format(vid))
+
+
+@pytest.fixture(scope="module")
+def config_facts(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+    """Get config facts for the duthost"""
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    return duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
+
+
+@pytest.fixture(scope="module")
+def intfs_for_test(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, tbinfo, config_facts):
+    """Get the interfaces that will be used in our ARP test"""
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    asic = duthost.asic_instance(enum_frontend_asic_index)
+    mg_facts = asic.get_extended_minigraph_facts(tbinfo)
+    external_ports = [p for p in list(mg_facts['minigraph_ports'].keys()) if 'BP' not in p]
+    ports = list(sorted(external_ports, key=lambda item: int(item.replace('Ethernet', ''))))
+    po1 = None
+    po2 = None
+
+    is_storage_backend = 'backend' in tbinfo['topo']['name']
+
+    if tbinfo['topo']['type'] == 't0':
+        if is_storage_backend:
+            vlan_sub_intfs = mg_facts['minigraph_vlan_sub_interfaces']
+            intfs_to_t1 = [_['attachto'].split(constants.VLAN_SUB_INTERFACE_SEPARATOR)[0] for _ in vlan_sub_intfs]
+            ports_for_test = [_ for _ in ports if _ not in intfs_to_t1]
+
+            intf1 = ports_for_test[0]
+            intf2 = ports_for_test[1]
+        else:
+            if 'PORTCHANNEL_MEMBER' in config_facts:
+                portchannel_members = []
+                for _, v in list(config_facts['PORTCHANNEL_MEMBER'].items()):
+                    portchannel_members += list(v.keys())
+                ports_for_test = [x for x in ports if x not in portchannel_members]
+            else:
+                ports_for_test = ports
+
+            # Select two interfaces for testing which are not in portchannel
+            intf1 = ports_for_test[0]
+            intf2 = ports_for_test[1]
+    else:
+        # Select first 2 ports that are admin 'up'
+        intf_status = asic.show_interface(command='status')['ansible_facts']['int_status']
+
+        intf1 = None
+        intf2 = None
+        for a_port in ports:
+            if intf_status[a_port]['admin_state'] == 'up':
+                if intf1 is None:
+                    intf1 = a_port
+                elif intf2 is None:
+                    intf2 = a_port
+                else:
+                    break
+
+        if intf1 is None or intf2 is None:
+            pytest.skip("Not enough interfaces on this host/asic (%s/%s) to support test." % (duthost.hostname,
+                                                                                              asic.asic_index))
+        po1 = get_po(mg_facts, intf1)
+        po2 = get_po(mg_facts, intf2)
+
+        if po1:
+            asic.config_portchannel_member(po1, intf1, "del")
```

Comment:
Why do we need to remove the port from portchannel?

Reply:
Removed.